website/docs/r/ai_foundry_project.html.markdown

```diff
@@ -71,10 +71,40 @@ resource "azurerm_ai_foundry" "example" {
   }
 }
 
+resource "azurerm_user_assigned_identity" "example" {
+  location            = azurerm_resource_group.example.location
+  name                = "identity-example"
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_role_assignment" "example" {
+  scope                = azurerm_resource_group.example.id
+  role_definition_name = "Reader"
+  principal_id         = azurerm_user_assigned_identity.example.principal_id
+}
+
+resource "azurerm_key_vault_access_policy" "example2" {
+  key_vault_id = azurerm_key_vault.example.id
+  tenant_id    = azurerm_user_assigned_identity.example.tenant_id
+  object_id    = azurerm_user_assigned_identity.example.client_id
+
+  key_permissions = [
+    "Create",
+    "Get",
+  ]
+}
+
+
```

Comment:
since `identity` and `primary_user_assigned_identity` are optional properties, we don't need to include config for them in the example
```suggestion
```